libobs/obs-module.c

```diff
@@ -273,6 +273,10 @@ static void load_all_callback(void *param, const struct obs_module_info *info)
 {
 	obs_module_t *module;
 
+	if (!os_is_obs_plugin(info->bin_path))
+		blog(LOG_WARNING, "Skipping module '%s', not an OBS plugin",
+		     info->bin_path);
```

Comment:
Do you lost a "return" here ?

Reply:
@notr1ch

Reply:
I created a PR https://github.com/obsproject/obs-studio/pull/5625

Reply:
Apparently so, not sure how I missed that when it's the entire point of the PR 🤦‍♂️